apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,14 +1,14 @@
 ack_generate_info:
-  build_date: "2021-06-22T20:41:21Z"
-  build_hash: 38e309d07369b1e33250473084f91db3894c4379
-  go_version: go1.16.4 darwin/amd64
```

Comment:
this keeps changing between 1.14 and 1.16 according to the developer machine

@A-Hilaly is all this metadata needed? especially the checksums
arent the runtime versions and code gen versions commit ids sufficient?

Reply:
I would say that the Go version is not very important, but it might eventually be helpful for debugging if anything goes wrong in the future (with Go stdlibs/runtime)